build.py

```diff
@@ -380,22 +384,25 @@ def tensorrt_cmake_args():
 
 
 def tensorflow_cmake_args(ver, images):
-    # If a specific TF image is specified use it, otherwise pull from
-    # NGC.
-    image_name = "tensorflow{}".format(ver)
-    if image_name in images:
-        image = images[image_name]
+    # If jetson build is specified do not use docker images
+    if FLAGS.jetson_build:
+        opt_dir = "/opt" if FLAGS.opt_dir is None else FLAGS.opt_dir
```

Comment:
Instead of --opt-dir add a new options that are like --image in that they provide a way to give library and include paths for certain parts of the build. Add --library-paths and --include-paths that are used in the same way as --image. For example --library-paths=tensorflow1:/opt/tritonserver/backends/tensorflow1

Reply:
For now we do not have any include paths that we need to specify in the build, we can add them in as needed. Added the library-paths arg and remove opt-dir